src/test/java/com/actionlistapi/services/ProcessInstanceStatusConverterSpec.java

```diff
@@ -0,0 +1,42 @@
+package com.actionlistapi.services;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.junit4.SpringRunner;
+import com.actionlistapi.pojo.ProcessInstanceStatus;
+
+@RunWith(SpringRunner.class)
+public class ProcessInstanceStatusConverterSpec {
```

Comment:
The convention in Java is to name Unit tests as `*Test.java` (so, `ProcessInstanceStatusConverterTest` in this case)